docs/tutorials/elemental_tutorial.md

```diff
@@ -0,0 +1,373 @@
+# Introduction
+
+In this tutorial Kubernetes' core concepts Pods, Services, Routes and ReplicationControllers and their YAML representations are discussed.
+
+In these examples, a apache http server is started from container image included in openshift by default and the server is exposed at myservice-my-project-with-unique-name.rahtiapp.fi.
```

Comment:
This seems like a shortcoming in mkdocs. Is it possible to disable automatic url link creation, since, anyhow, bare links in markdown is a violation according to markdown linter.

Reply:
Actually, I'll just make it `<myservice>.rahtiapp.fi`

Reply:
> Actually, I'll just make it `<myservice>.rahtiapp.fi`

This is a good way of handling this.